docs/api/python/ndarray.md

```diff
@@ -17,7 +17,7 @@ This document lists the routines of the *n*-dimensional array package
 
 The `NDArray` API, defined in the `ndarray` (or simply `nd`) package, provides
 imperative tensor operations on CPU/GPU.
-A `NDArray` represents a multidimensional, fixed-size homogenous array.
+An `NDArray` represents a multi-dimensional, fixed-size homogenous array.
```

Comment:
'An' is better than 'A' since NDArray starts off with a vowel like pronunciation.